### ISSUE TITLE
if vs ternary

### ISSUE THREAD

**Issue by bsonntag** [Hidepixel/guides#1]
Should we allow [ternaries](https://en.wikipedia.org/wiki/%3F:) or should we prefer `if` statements?

**Comment by bsonntag** [comment]
bump @DanielMor, @Grundge and @Mickael24

**Comment by Mickael24** [comment]
this will very subjective and will depend on what you want to do in the else :). The level of readability I prefer if.

**Comment by Grundge** [comment]
Use ternaries whenever possible.

**Comment by bsonntag** [comment]
@Grundge I think we should avoid them... Although they are concise, they are less readable than `if`s.

Example:

```js
function buildImageUrl(url, size) {
    var base = '/api/files/';
    return url.match(/http/) ? url :
        size ? base + url + '?size=' + size :
        base + url;
}
```

**Comment by Grundge** [comment]
I didn't consider concatenation of multiple ternaries :) I would accept a ternary ONLY once, and if it were more, I would adopt if/else due to readability.

**Comment by bsonntag** [comment]
bump @ruimfernandes and @ana-lopes

**Comment by ana-lopes** [comment]
I agree with @Grundge. I personally only use ternaries once otherwise it loses its readability.

**Comment by ruimfernandes** [comment]
I don't like ternaries... :scream:

**Comment by bsonntag** [comment]
Alright, so should the rule be "don't chain multiple ternaries"?
